v2/pkg/cli/executor.go

```diff
@@ -44,22 +44,59 @@ import (
 var (
 	mirrorlongDesc = templates.LongDesc(
 		` 
-		Create and publish user-configured mirrors with a declarative configuration input.
-		used for authenticating to the registries. 
+		Mirror OCP Release, operator catalog and additional images using a declarative configuration file as an input. 
 
-		The podman location for credentials is also supported as a secondary location.
+		The command line uses a declarative configuration file as the input to discover from where to get the container images and copy them to the destination specified in the command line. 
 
-		1. Destination prefix is docker:// - The current working directory will be used.
-		2. Destination prefix is oci:// - The destination directory specified will be used.
+		There are three workflows available in oc-mirror currently:
+			
+			- mirrorToDisk - pulls the container images from the source specified in the image set configuration and pack them into a tar archive on disk (local directory).
+			- diskToMirror - copy the containers images from the tar archive to a container registry (--from flag is required on this workflow).
+			- mirrorToDisk - copy the container images from the source specified in the image set configuration to the destination (container registry).
+
+		When specifying the destination on the command line, there are two prefixes available:
+
+			- file://<destination location> - used in mirrorToDisk: local mirror packed into a tar archive.
+			- docker://<destination location> - used in diskToMirror and mirrorToMirror: when the destination is a registry.
+
+		The default podman credentials location ($XDG_RUNTIME_DIR/containers/auth) is used for authenticating to the registries. The docker location for credentials is also supported as a secondary location.
+
+		The name of the directory used by oc-mirror as a workspace defaults to the name working-dir. The location of this directory depends on the following:
```

Comment:
```suggestion
		The name of the directory used by oc-mirror as a workspace defaults to the name 'working-dir'. The location of this directory depends on the following:
```

Reply:
Done :white_check_mark:

Thanks for the suggestion.